CloudAppSecurityDocs/troubleshooting-proxy-url.md

```diff
@@ -51,3 +51,6 @@ If the URL doesn't exactly match one of the replacement patterns (for example, `
 
 In case you do not recognize the remaining portion of the URL (e.g.: __myurl.com__.mcas.ms) as associated with any of your business apps, it's recommended that you investigate the issue further and consider blocking the URL to avoid any potential security risks.
```

Comment:
```suggestion
If you don't recognize the remaining portion of the URL (for example, __myurl.com__.mcas.ms) as associated with any of your business apps, it's recommended that you investigate the issue further and consider blocking the URL to avoid any potential security risks.
```